src/components/Dropdown/Dropdown.styles.js

```diff
@@ -315,8 +316,10 @@ const menu =
      * start when the menu position is fixed, and this is why in this case we define top:auto.
      */
     let top = insideOverflowWithTransformContainer ? "auto" : parentPositionData.bottom;
+    const translateY = getScrollableParent(controlRef?.current)?.scrollTop;
```

Comment:
1. run it only when `insideOverflowWithTransformContainer` is applied
2. when applying the scrollable container logic, the `insideOverflowWithTransformContainer` can simply apply `insideOverflowContainer` instead of applying the transform logic
3. in the example - make the dropdown scrollable in the middle (to explain scrolling before opening)